packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown.tsx

```diff
@@ -122,6 +122,7 @@ export class SelectWrapperDropdown {
       this.theme
     );
 
+    const part = 'p-select-wrapper-dropdown';
```

Comment:
Works with prefixing?

Reply:
Doesn't have to since it's just a part CSS selector. The naming can be anything.

Reply:
I've renamed it without `p-` prefix.